src/components/FilterGroup.tsx

```diff
@@ -51,6 +55,10 @@ export interface FilterGroupProps {
   showMoreLimit?: number
 }
 
+const builtInCssClasses: Readonly<FilterGroupCssClasses> = {
+  titleLabel: builtInCollapsibleLabelCssClasses.titleLabel,
```

Comment:
why do we need to make a `builtInCssClasses` variable for just this styling? can we just refer to `builtInCollapsibleLabelCssClasses.titleLabel` in the one place it's used below (for the `div`)?